src/mr-importer/loader.cpp

```
@@ -20,733 +22,1381 @@
     }
     else {
       MR_ERROR("Mesh has no material specified");
     }
 
-    graph.wait_for_all();
-
+    ASSERT(mesh.positions.size() == mesh.attributes.size());
     return mesh;
   }
+  else {
+    std::atomic_bool error = false;
```

Clang (14.0.6)

[warning] variable 'error' is not initialized

(cppcoreguidelines-init-variables)